sourcegraph/about
# Notebooks/code search updates

This closes #5198. It updates the Code Search page by adding a section for Notebooks between the "Search Engine Built for Code" and "Move Faster with Sourcegraph" sections.

- The copy is provided here: [copy doc](https://docs.google.com/document/d/1zOctM149X0OWiwcn3Udl8bIupS2ptiaBENxSJJiPL9o/edit). 
- Please note that the [GH issue](https://github.com/sourcegraph/about/issues/5198) requests that we only use the paragraph text. The section header comes from the screenshot at the bottom of the copy doc: "Document and explore code with Notebooks"
- The video is hosted on GCP at [link](https://storage.googleapis.com/sourcegraph-assets/notebooks/Notebooks_Capture_20s.mp4).
- The shadow under the video is emulated from a similar animation on the Code Insights page for consistency.

### Testing
- Navigate to `/code-search`
- Check copy for accuracy
- Ensure that the video autoplays and loops
- Assess the column order on mobile for this section (video on top, text second)
